changelog.txt

```diff
@@ -1,5 +1,20 @@
+Version 2.0.0
+-------------
+- Please note: ADAL 2.0.0 (released March 2020) is neither API-compatible nor cache-compatible the following versions:
```

Comment:
I remember hearing something like going straight to 3.0.0 Is that no longer happening?

Reply:
We'll stick with `2.0.0`